src/kaleidoscope/driver/keyscanner/ATMegaKeyScanner.h

```diff
@@ -137,12 +137,12 @@ class ATMegaScanner {
                    key_addr.col());
   }
 
-  static const ScannerDescription__ ScannerDescription_;
+  static const _KeyScannerDescription KeyScannerDescription_;
```

Comment:
Why is the description class instantiated?

Reply:
For no good reason - fixed!